src/Rule/NotEmpty.php

```diff
@@ -105,7 +105,10 @@ public function validate($value)
      */
     protected function isEmpty($value)
     {
-        return (is_string($value) && strlen($value) === 0) || $value === null;
+        return
+            (is_string($value) && strlen($value) === 0) ||
+            $value === null ||
+            (is_array($value) && count($value) === 0);
```

Comment:
I think it's good to add empty arrays here, but could you make this separate if statements? Thank you :)